### ISSUE TITLE
WebViewManager.bridgeSetup is not a function

### ISSUE THREAD

**Issue by WJWang** [alinz/react-native-webview-bridge#28]
Hi alinz,
I use the version react-native ^0.12.0, but I have a problem similar with #12 .
Here is my settings and testing code, is there any details may I have missed?
 
![issue](https://cloud.githubusercontent.com/assets/2817235/10509859/3d80b840-7362-11e5-9015-55e203188a39.png)
```
'use strict';

var React = require('react-native');
var WebViewBridgeNative = require('react-native-webview-bridge');

var myPageScripts = function() {
  function WebViewBridgeReady(cb) {
    if (window.WebViewBridge) {
      cb(window.WebViewBridge);
      return;
    }
    function handler() {
      document.removeEventListener('WebViewBridge', handler, false);
      cb(window.WebViewBridge);
    }
    document.addEventListener('WebViewBridge', handler, false);
  }
  WebViewBridgeReady(function (WebViewBridge) {
    WebViewBridge.send("Hello this is me calling from web page");
  });
};

//convert function definition into string
myPageScripts = `(${myPageScripts.toString()}());`;

var {
  AppRegistry,
  Component
} = React;

class testingBridge extends Component {
  constructor(props) {
    super(props);
    this.once = true;
  }

  componentDidMount() {
    var myWebViewBridgeRef = this.refs.myWebViewBridge;

    var onMessage = function(message) {
      console.log("Received message", message);
    };
    myWebViewBridgeRef.onMessage(onMessage);

    myWebViewBridgeRef.injectBridgeScript();

    //this opens up the printer window dialoge
    setTimeout(() => {
      myWebViewBridgeRef.print();
    }, 5000);
  }

  onNavigationStateChange(navState) {
    var myWebViewBridgeRef = this.refs.myWebViewBridge;
    if (this.once) {
      this.once = false;
      setTimeout(() => {
        myWebViewBridgeRef.evalScript(myPageScripts);
      }, 1000);
    }

    console.log(navState.url);
  }

  render() {
    var url = 'http://google.com';

    return (
      <WebViewBridgeNative
        ref="myWebViewBridge"
        onNavigationStateChange={this.onNavigationStateChange.bind(this)}
        url={url}
        style={{flex: 1}}/>
    );
  }
}

AppRegistry.registerComponent('testingBridge', () => testingBridge);
```

**Comment by WJWang** [comment]
After I pull all the dependency files out of the upper directory, it work well.
![2015-10-15 6 21 56](https://cloud.githubusercontent.com/assets/2817235/10511114/a3fa94cc-7369-11e5-9c1b-59d8d5e467d4.png)

**Comment by gre** [comment]
that doesn't look a good solution though. Ideally `react-native-webview-bridge` should provide an xcodeproj project like in all other react-native lib

**Comment by alinz** [comment]
@gre OK, I've heard you :P, I have updated the project to support that. Enjoy! :beer:

**Comment by gre** [comment]
:+1: awesome

**Comment by johanmontan** [comment]
Had the same issues even thought I followed the instructions in [Linking Libraries](http://facebook.github.io/react-native/docs/linking-libraries-ios.html) as it was my first react-native project and didn't have a clue where to begin.
I realised that the this library didn't build properly so I commented out the code that didn't work. In this case RCTLogMustFix(). 
(Running react-native 0.15.0 within Xcode 7.1).
